portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,149 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+@import url('https://fonts.googleapis.com/css?family=Raleway&display=swap');
+
+*{
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  list-style: none;
+  text-decoration: none;
+  font-family: 'Raleway', sans-serif;
+}
+
+body{
+   background-color: #f3f5f9;
+}
+
+.wrapper{
+  display: flex;
+  position: relative;
+}
+
+/* -------------- Side Bar --------------*/
+
+.wrapper .sidebar{
+  width: 280px;
+  height: 100%;
+  background: #6767b5;
+  padding: 30px 0px;
+  position: fixed;
+}
+
+/* -------------- Side Bar Avatar --------------*/
+
+.wrapper .sidebar .avatar {
+    position: relative;
+    overflow: hidden;
+    margin: 55px;
+    width: 170px;
+    height: 170px;
+    -webkit-border-radius: 50%;
+    -moz-border-radius: 50%;
+    -ms-border-radius: 50%;
+    border-radius: 50%;
+}
+.wrapper .sidebar .avatar img {
```

Comment:
Hmm, just realized you have a div wrapping the img. It might be easier for you to remove the div and just do the image with a display: block.